static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -30,7 +34,11 @@ define([
                 var oldHref = link.attr('href');
                 var props = {
                     page: window.location,
-                    width: window.innerWidth
+                    width: window.innerWidth,
```

Comment:
So, you're just grabbing the creativeID properties? Could you not just do

    ads : _.map(dfp.getCreativeIDs(), 'creativeID'),

- or, if you don't want to work with string keys -

    ads: _.map(dfp.getCreativeIDs(), function (creativeData) {return creativeData.creativeId;}